### ISSUE TITLE
git icon not showing problem

### ISSUE THREAD

**Issue by CodingInvoker** [romkatv/powerlevel10k#1009]
Instead of showing the cute git cat icon, mine is always showing `git` literals as the screenshot below:
![image](https://user-images.githubusercontent.com/71058225/92996050-10aefb00-f53b-11ea-842f-5f8a9a31d6b5.png)

After checking this this [issue](https://github.com/Powerlevel9k/powerlevel9k/issues/1166) and chatting with @romkatv I got confirmed that this is indeed as expected since I am using github enterprise.

Long story short, (correct me if I am wrong)in order to show the cute cat we need our git remote url to contain `github` segment. However I suppose a lot of folks are working with Github Enterprise like me, meaning that the remote url would be something like `git.mydopecompany.com` but not `github.com`. 

Can we plz have this cute cat icon enabled for github enterprise users as well?

**Comment by romkatv** [comment]
For my education, why do you want that icon?

**Comment by CodingInvoker** [comment]
For a consistent user experience I guess :D  cuz github enterprise is still github with cat icon

**Comment by romkatv** [comment]
> For a consistent user experience I guess :D cuz github enterprise is still github with cat icon

There are a few things you can do.

One option is to do nothing and change your expectations. Expect GitHub logo to be shown when and only when the remote is on github.com. You might even be able to rationalize why it's better than showing GitHub logo for both github.com and git.mydopecompany.com (it allows you to distinguish between the two).

Another option is to turn this icon off completely. You can do this by running `p10k configure` and choosing "Few Icons". This option disables decorative icons (those that convey no or little information), which includes the github/git icon and the branch icon.

Lastly, you can manually define which icon gets shown for different remotes. To do this, you'll first need to disable the builtin icon:

```zsh
typeset -g POWERLEVEL9K_VCS_VISUAL_IDENTIFIER_EXPANSION=
```

Then modify `my_git_formatter`:

```diff
--- a/config/p10k-classic.zsh
+++ b/config/p10k-classic.zsh
@@ -372,6 +372,13 @@
     fi
 
     local res
+
+    case $VCS_STATUS_REMOTE_URL in
+      *github.com*)            res+=$'\uF113 ';;
+      *git.mydopecompany.com*) res+=$'\uF113 ';;
+      *)                       res+=$'\uF1D3 ';;
+    esac
+
     local where  # branch or tag
     if [[ -n $VCS_STATUS_LOCAL_BRANCH ]]; then
       res+="${clean}${(g::)POWERLEVEL9K_VCS_BRANCH_ICON}"
```

Adjust this logic as you like. E.g., you can display your company logo (or something that resembles it) when the remote is on git.mydopecompany.com.

**Comment by ShaneTian** [comment]
```
typeset -g POWERLEVEL9K_VCS_VISUAL_IDENTIFIER_EXPANSION=
```
⬆️ It does not work.
When I added the code about res in `~/.p10k.zsh`, two icons appeared in git repository ⬇️
![image](https://user-images.githubusercontent.com/42370681/94549258-82e64600-0284-11eb-8e10-9d0dfb556d91.png)

**Comment by romkatv** [comment]
> ```
> typeset -g POWERLEVEL9K_VCS_VISUAL_IDENTIFIER_EXPANSION=
> ```
> 
> ⬆️ It does not work.

What is the output of the following command?

```zsh
typeset -p POWERLEVEL9K_VCS_VISUAL_IDENTIFIER_EXPANSION
```

**Comment by ShaneTian** [comment]
> ```shell
> typeset -p POWERLEVEL9K_VCS_VISUAL_IDENTIFIER_EXPANSION
> ```

typeset: no such variable: POWERLEVEL9K_VCS_VISUAL_IDENTIFIER_EXPANSION

**Comment by romkatv** [comment]
> ```shell
> ```shell
> typeset -p POWERLEVEL9K_VCS_VISUAL_IDENTIFIER_EXPANSION
> ```
> 
> 
> 
> typeset: no such variable: POWERLEVEL9K_VCS_VISUAL_IDENTIFIER_EXPANSION
> ```

Do you understand what it says?

**Comment by ShaneTian** [comment]
> > ```shell
> > typeset -p POWERLEVEL9K_VCS_VISUAL_IDENTIFIER_EXPANSION
> > ```
> > 
> > 
> > typeset: no such variable: POWERLEVEL9K_VCS_VISUAL_IDENTIFIER_EXPANSION
> 
> Do you understand what it says?

I think the variable `POWERLEVEL9K_VCS_VISUAL_IDENTIFIER_EXPANSION` was not set successfully. But when I set the variable to empty, it still does not take effect
![image](https://user-images.githubusercontent.com/42370681/94551052-51bb4500-0287-11eb-80a5-e3857848f1ef.png)

**Comment by romkatv** [comment]
You need to set it in `~/.p10k.zsh` together with all other parameters.

**Comment by ShaneTian** [comment]
> You need to set it in `~/.p10k.zsh` together with all other parameters.

Yes, I dit it. In `~/.p10k.zsh`:
![image](https://user-images.githubusercontent.com/42370681/94552197-1457b700-0289-11eb-825c-dae948ab0fa7.png)
After the modification, I tried two ways:
- Open a new tab
- source `~/.p10k.zsh`

I found that none of them worked, and two icons appeared in git repository.
Is there anything wrong?

**Comment by romkatv** [comment]
You need to set this parameter where all other parameters are set. The file contains many examples of `typeset -g POWERLEVEL9K_...`. Notice how those lines are very similar to the one you want to add. So add it next to all those lines. Do not add it within a function that happens to not have any similar lines. (Or, alternatively, learn the basics of zsh so that you understand what's going on.)
